rclpy/rclpy/duration.py

```diff
@@ -77,5 +82,9 @@ def from_msg(cls, msg):
             raise TypeError('Must pass a builtin_interfaces.msg.Duration object')
         return cls(seconds=msg.sec, nanoseconds=msg.nanosec)
 
+    @classmethod
+    def Infinite(cls):
+        return cls(nanoseconds=_rclpy.RMW_DURATION_INFINITE)
```

Comment:
@emersonknapp nit: considering `Duration` objects are immutable, to have a single `Infinite` instance would be best.

Reply:
sure